src/Core/Crawl/controlpos.h

```diff
@@ -14,7 +14,18 @@
 namespace CRAWL {
 
 /**
- * @brief The ControlPos class
+ * @brief The Refresh enum Lists the shapes to convert the group object to.
+ */
+enum Refresh {
+    CIRCLE,
+    SPHERE,
+    SQUARE,
+    CUBE,
+    LINE
```

Comment:
remove not supported modes 
```suggestion
enum Refresh {
    CIRCLE,
    SQUARE,
    LINE
```